src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -215,6 +217,110 @@ public static long Flt2LngOvf(float val)
             return ThrowIntOvf();
         }
 
+#if ARM
+        private const string RuntimeLibrary = "[MRT]";
+
+        [RuntimeImport(RuntimeLibrary)]
```

Comment:
This should be `RuntimeImport("RhpIDiv")` (and same change everywhere else).

Reply:
Or `[RuntimeImport(RuntimeLibrary, "RhpIDiv")]` to match how other RuntimeImports are done in CoreLib (the difference does not matter for CoreRT - both will work fine for CoreRT).

Reply:
@jkotas @MichalStrehovsky Thanks to both of you!